### PR TITLE
Add playonlinux derivation.

### PR DESCRIPTION
**Pull request by a1russell**
I'd like to add a derivation for [PlayOnLinux](https://www.playonlinux.com). This is my first contribution here, so please let me know if there's anything I should change. My nixos-version is `15.08pre67375.f423ba3 (Dingo)`.

_Edit: the below problem is solved now, so you can now ignore it._

Actually, the derivation as it is has some issues, and I do need some help working them out. After installing and launching playonlinux, I receive errors both via a GUI dialog and on the console.

The first GUI dialog error I receive:

> PlayOnLinux is unable to find 32bits OpenGL libraries.
> You might encounter problem with your games
> [OK]

Looking at the console, there is this exception:

> [Check_OpenGL] Warning:
> /nix/store/<hash>-playonlinux-4.2.8/share/playonlinux/bash/check_gl: line 42: /home/<me>/.PlayOnLinux//tmp/check_dd_x86: No such file or directory

Alright, so I click OK on the GUI dialog, and I get a second, very similar dialog:

> PlayOnLinux is unable to find 32bits OpenGL libraries.
> You might encounter problem with your games
> [OK]

And also another exception on the console:

> [Check_OpenGL] Warning:
> /nix/store/<hash>-playonlinux-4.2.8/share/playonlinux/bash/check_gl: line 42: /home/<me>/.PlayOnLinux//tmp/check_dd_amd64: No such file or directory

For what it's worth, those `check_dd_*` files definitely exist. I can't tell how to make them work though.

Any help?

### CODE REVIEW

**Comment by fazo96** [comment]
Hey, I'm interested about playonlinux too!

I think you need to enable 32 bit opengl libraries as explained in the [wiki page about steam](https://nixos.org/wiki/Steam): just try putting `hardware.opengl.driSupport32Bit = true` in your configuration. I'll try too with your derivation if I have the time

**Comment by a1russell** [comment]
Unfortunately I do already have `hardware.opengl.driSupport32Bit` enabled, and 64 bit is having trouble, too, besides.

File: pkgs/applications/misc/playonlinux/default.nix [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        --prefix PATH : ${p7zip}/bin \
+        --prefix PATH : ${python}/bin \
+        --prefix PATH : ${unzip}/bin \
+        --prefix PATH : ${wget}/bin \
+        --prefix PATH : ${wine}/bin \
+        --prefix PATH : ${xdg-user-dirs}/bin \
+        --prefix PATH : ${xterm}/bin
+    done
+  '';
+
```
Comment (jagajaga) [inline pkgs/applications/misc/playonlinux/default.nix]: Don't you want to be a maintainer?

**Comment by jagajaga** [comment]
Please rename commit's names following our guide in `CONTRIBUTING.md`.

**Comment by a1russell** [comment]
@jagajaga I renamed the commit. I'm not sure about what being a maintainer entails exactly. Could I be a joint maintainer along with the folks that are more used to this? Also I wasn't sure what the namespacing of maintainers meant. I did see to add my info in maintainers.nix if I desired that.

Any idea on the problem I'm having (I am guessing with libGL), though?

**Comment by a1russell** [comment]
I think I just figured it out. The two files in question are $out/share/playonlinux/bin/check_dd_amd64.bz2 and $out/share/playonlinux/bin/check_dd_x86.bz2. Bunzipped, these are ELF files. The nix builder fixupPhase runs a command called `patchelf` on any ELF file it detects; however, since these two files are bzipped, `patchelf` was not detecting them. I shall bunzip them before fixupPhase and bzip them back afterwards.

**Comment by a1russell** [comment]
Pushed the fix and added myself as the maintainer, too.

**Pull request merged**